pkg/distributor/distributor.go

```diff
@@ -2079,6 +2080,34 @@ func (cm *labelValuesCardinalityConcurrentMap) toResponse(approximateFromZonesFu
 // ActiveSeries queries the ingester replication set for active series matching
 // the given selector. It combines and deduplicates the results.
 func (d *Distributor) ActiveSeries(ctx context.Context, matchers []*labels.Matcher) ([]labels.Labels, error) {
+	res, err := d.deduplicateActiveSeries(ctx, matchers, false)
+	if err != nil {
+		return nil, err
+	}
+
+	deduplicatedSeries, fetchedSeries := res.result()
+
+	reqStats := stats.FromContext(ctx)
+	reqStats.AddFetchedSeries(fetchedSeries)
```

Comment:
[nit]

why not keep it as it was before?
```suggestion
	deduplicatedSeries := res.result()

	reqStats := stats.FromContext(ctx)
	reqStats.AddFetchedSeries(len(deduplicatedSeries))
```

Reply:
I had to add the fetchedSeries return value for the metrics case , to be able to provide accurate statistics there, so it seemed more consistent to change it here as well. Want me to change it back ? Don't have a strong preference.

Reply:
I think it's a tad more readable at the call site to have one return value and take the length of that, but I don't have a strong opinion, so we can also leave as is.